libr/include/r_cmd.h

```diff
@@ -113,14 +139,24 @@ R_API int r_core_plugin_add(RCmd *cmd, RCorePlugin *plugin);
 R_API int r_core_plugin_check(RCmd *cmd, const char *a0);
 R_API int r_core_plugin_fini(RCmd *cmd);
 
-/* review api */
 R_API RCmd *r_cmd_new(void);
 R_API RCmd *r_cmd_free(RCmd *cmd);
 R_API int r_cmd_set_data(RCmd *cmd, void *data);
-R_API int r_cmd_add(RCmd *cmd, const char *command, const char *desc, r_cmd_callback(callback));
+R_API int r_cmd_add(RCmd *cmd, const char *command, RCmdCb callback);
 R_API int r_core_del(RCmd *cmd, const char *command);
 R_API int r_cmd_call(RCmd *cmd, const char *command);
 R_API int r_cmd_call_parsed_args(RCmd *cmd, RCmdParsedArgs *args);
+R_API RCmdDesc *r_cmd_get_root(RCmd *cmd);
+R_API RCmdDesc *r_cmd_get_desc(RCmd *cmd, const char *cmd_identifier);
+
+/* RCmdDescriptor */
+R_API RCmdDesc *r_cmd_desc_inner_new(RCmd *cmd, RCmdDesc *parent, const char *name, const char *help);
```

Comment:
actually, since I'm currently not using this `help` (and the others) anywhere I may as well just remove these. I think we are going to change these API a bit anyway once we agree on what to have as help. @karliss are you ok with me removing the `help` field and `help` in argument?

Reply:
Sounds good..